action.yml

```diff
@@ -567,13 +571,28 @@ runs:
         echo "No builder was able to build the provided application source. Please visit the following page for more information on supported platform versions: https://aka.ms/SourceToCloudSupportedVersions"
         exit 1
 
+    - name: Convert build arguments to string
+      if: ${{ inputs.buildArguments != '' }}
```

Comment:
Let's update this conditional so that we're only running it when the user is providing (or we find) a `Dockerfile` -- in this case, we're far enough along in the action that the `CA_GH_ACTION_DOCKERFILE_PATH` environment variable will have been set if a `Dockerfile` was provided or found, so we can add a check for that being not empty (along with the check if the user provided application source):

```
if: ${{ inputs.dockerfileBuildArguments != '' &&  inputs.appSourcePath != '' && env.CA_GH_ACTION_DOCKERFILE_PATH != '' }}
```

This ensures that we're only going to be reading this argument in the case that it'll be used.